dataset/dictionary/locations.json5

```
@@ -1,11 +1,36 @@
 [
   {
-    en: "",
-    ja: "",
-    zhCN: "",
-    pronunciationJa: "",
-    notes: "",
-    notesZh: "",
-    tags: [ "location" ],
+    en: "Jarilo-VI",
+    ja: "ヤリーロシックス",
```

Oops, you didn't have to change `ja`. "6" is expressed in Roman figure (Ⅵ) in Japanese as well as English and Chinese.
I expressed "6" in kana ("シックス") as a `pronunciationJa`.

```suggestion
    ja: "ヤリーロ-Ⅵ",
    pronunciationJa: "ヤリーロシックス",
```